datalake/backend/tests/test_check_stac_metadata.py

```diff
@@ -47,7 +47,7 @@
     },
 }
 
-BUCKET_NAME = f"linz-geospatial-data-lake-{ENV}"
+BUCKET_NAME = ResourceName.STORAGE_BUCKET_NAME.value
```

Comment:
I would just inline (remove) the constant since it's just a proxy for another constant now.